chromium_src/BUILD.gn

```diff
@@ -388,14 +388,19 @@ source_set("chrome_spellchecker") {
       "//chrome/browser/spellchecker/spellcheck_factory.h",
       "//chrome/browser/spellchecker/spellcheck_hunspell_dictionary.cc",
       "//chrome/browser/spellchecker/spellcheck_hunspell_dictionary.h",
-      "//chrome/browser/spellchecker/spellcheck_language_blocklist_policy_handler.cc",
-      "//chrome/browser/spellchecker/spellcheck_language_blocklist_policy_handler.h",
-      "//chrome/browser/spellchecker/spellcheck_language_policy_handler.cc",
-      "//chrome/browser/spellchecker/spellcheck_language_policy_handler.h",
       "//chrome/browser/spellchecker/spellcheck_service.cc",
       "//chrome/browser/spellchecker/spellcheck_service.h",
     ]
 
+    if (is_win || is_linux || is_chromeos) {
```

Comment:
maybe just `if (!is_mac)`? a bit weird to have `is_chromeos` in our build files since we don't target that platform.

Reply:
true... I copied from the upstream change but if (!is_mac) would be more straightforward for our use.